.github/workflows/ci.yml

```diff
@@ -44,23 +58,46 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
-      - name: Build
-        run: pnpm build
+      #- name: Build
+      #  run: pnpm -C packages/hardhat build
+
+      # - name: Lint
+      #   run: pnpm lint
 
-      - name: Lint
-        run: pnpm lint
+      # - name: Test
+      #   run: pnpm -filter '!@sifi/hardhat' packages/frontend test
 
-      - name: Test
-        run: pnpm -filter '!@sifi/hardhat' packages/frontend test
+      # - name: Test hardhat
+      #   env:
+      #     RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
+      #   if: ${{ env.MAINNET_RPC_URL != null }}
+      #   run: pnpm -C packages/hardhat test
 
-      - name: Test hardhat
-        env:
-          RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
-        if: ${{ env.MAINNET_RPC_URL != null }}
-        run: pnpm -C packages/hardhat test
+      - name: Foundry forge build
+        run: pnpm -C packages/hardhat forge-build
 
-      - name: Test foundry
-        env:
-          RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
-        if: ${{ env.MAINNET_RPC_URL != null }}
-        run: pnpm -C packages/hardhat forge test -vv
+      - name: Foundry forge test
+        run: pnpm -C packages/hardhat forge-test
+
```

Comment:
Many steps were commented out to make testing Slither on Actions faster. You can remove these comments now